docs/user_guide/nvflare_cli.rst

```diff
@@ -4,6 +4,20 @@
 NVFlare CLI
 ###########################
 
-In version 2.2, the commands for NVIDIA FLARE have been consolidated to be under the ``nvflare`` command for
+The commands for NVIDIA FLARE have been consolidated to be under the ``nvflare`` command for
 better ease of use. This includes the FL Simulator, the POC command, ``provision``, and preflight check, all of
-which are explained in more detail in their own sections in the :ref:`user_guide`.
\ No newline at end of file
+which are explained in more detail in their own sections:
+
+    - :ref:`fl_simulator`
+    - :ref:`poc_command`
+    - :ref:`provision_command`
+    - :ref:`preflight_check`
+    - :ref:`dashboard_api`
+
+.. toctree::
+   :maxdepth: -1
+   :hidden:
+
+   nvflare_cli/fl_simulator
+   nvflare_cli/poc_command
+   nvflare_cli/preflight_check
```

Comment:
should we move the documentation section regarding provision command and dashboard command inside this as well?

Reply:
I'll move them here in the toctree